### ISSUE TITLE
::ffff prefix is not stripped out

### ISSUE THREAD

**Issue by taylorskalyo** [Lejo1/vps_blocker#7]
Got this (ip and player name obfuscated with `127.0.0.1` and `playername` respectively):
```
2022-05-14 01:43:51: ERROR[CurlFetch]: http://v2.api.iphub.info/ip/::ffff:127.0.0.1 not found (Timeout was reached) (response code 0)
2022-05-14 01:43:51: ERROR[Server]: Failed to parse json data * Line 1, Column 1
2022-05-14 01:43:51: ERROR[Server]:   Syntax error: value, object or array expected.
2022-05-14 01:43:51: ERROR[Server]: data: ""
2022-05-14 01:43:51: ERROR[Server]: vps_blocker: Callback-Error Incorrect iphub request! while checking playername [::ffff:127.0.0.1]!
```

https://github.com/Lejo1/vps_blocker/issues/3#issuecomment-611640939 suggests that iphub filters out the `::ffff` prefix (is that documented somewhere?), but that doesn't appear to be the case here.

Is it possible for vps_blocker to filter out the prefix?

**Comment by taylorskalyo** [comment]
Maybe something like this? https://github.com/Lejo1/vps_blocker/pull/8

**Comment by Lejo1** [comment]
Iphub does strip out the ::ffff: prefix as described in #3 
I think your problem is that "::ffff:127.0.0.1" is a local ip.
There is no way for iphub to check them, so stripping the prefix out in the mod shouldn't change anything.

Are you connecting via some proxy or vpn on your server to your server?
Or are you using some kind of udp-reverse proxy? (Not sure if this is even possible...)

**Comment by taylorskalyo** [comment]
The original address in the logs was my residential ip. I replaced it with 127.0.0.1 here for privacy.

I tested the iphub API call myself, and you are right, it does seem to strip out the prefix (both 127.0.0.1 and the original ip).
```
$ curl http://v2.api.iphub.info/ip/::ffff:127.0.0.1 -H "X-Key: MTc0Njg6VTk3Mk9wdmxpcVdoNTJGbkxwR2VKUmVGa0ZuUGp0bTg="
{"ip":"127.0.0.1","countryCode":"ZZ","countryName":"ZZ","asn":0,"isp":"Private or unnanounced IP","block":0,"hostname":"127.0.0.1"}
```

My new thought is that this is probably the issue:
> (Timeout was reached)

The mod could potentially implement a retry mechanism, but that might be overkill.

Thanks for taking the time to respond! I'm going to go ahead and close this issue.
